x-pack/test/functional/apps/canvas/feature_controls/canvas_security.ts

```diff
@@ -217,34 +217,22 @@ export default function ({ getPageObjects, getService }: FtrProviderContext) {
         await security.user.delete('no_canvas_privileges_user');
       });
 
-      it(`returns a 404`, async () => {
+      it(`returns a 403`, async () => {
         await PageObjects.common.navigateToActualUrl('canvas', '', {
           ensureCurrentUrl: false,
           shouldLoginIfPrompted: false,
         });
         const messageText = await PageObjects.common.getBodyText();
-        expect(messageText).to.eql(
-          JSON.stringify({
-            statusCode: 404,
-            error: 'Not Found',
-            message: 'Not Found',
-          })
-        );
+        expect(messageText).to.contain('You do not have permission to access the requested page');
```

Comment:
Nit: we search for this message text in several test files -- it would probably be better to abstract this out to a method such as `PageObjects.security.expectForbiddenPage()`.

Reply:
To be concise, how about renaming the two existing checks `expectForbidden` and `expectNotFound` to `expectForbiddenJson` and `expectNotFoundJson` respectively? That more clearly shows that one is explicitly expecting and checking for a JSON response while the other expects a real "page".

Reply:
Sounds reasonable to me!

Reply:
Doh, I forgot that there's no reason to keep an old version of `expectForbidden ` around. When I was looking at it I was looking in `master`. But in this PR it has been completely rewritten to the new use-case. So there's no need to have its name reflect the content-type after all. I'll push a commit that addresses your concern ASAP.

Reply:
Fixed